### ISSUE TITLE
28532959: OS Keyboard extension loses height (turns to 0) when using autolayout automatic sizing in some cases, only on iOS 10

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#15992]
#### Description

Summary:
When creating a keyboard extension and not specifying a concrete height for its components but instead anchoring them to the view/inputView so that in theory the system will determine their height based on environment and orientation, in some situations that height instead turns into 0 and the keyboard is crushed (with the exception of anything that has a concrete height such as a self sized label or button).

This only seems to occur on iOS 10. On iOS 9, the child views resized correctly to fit the default automatic keyboard height.

Steps to Reproduce:
I have attached a basic sample project which illustrates this issue. It is just the custom keyboard template with one extra view in it anchored to the default "next keyboard" button. You can also find detailed instructions and screenshots in this github repo: https://github.com/DimaVartanian/keyboard-extension-height-bug

Basically:
1. Create a custom keyboard using the Xcode template. There should be 1 subview anchored to the bottom left (the "next keyboard" button).
2. Create another view and anchor its top to the top of the root view and its bottom to the top of the button.

Expected Results:
You would expect to see your new view stretched vertically to fill the space between the top border and the button.

Actual Results:
Works fine in iOS 9. In iOS 10, the view instead sizes down to 0 and all that remains is the "next keyboard" button which has an intrinsic height.
## 

Product Version: Xcode 8.0 (8A218a)
Created: 2016-09-29T00:41:44.899170
Originated: 2016-09-28T00:00:00
Open Radar Link: http://www.openradar.me/28532959
